external/easylogging++/easylogging++.h

```diff
@@ -203,16 +203,17 @@ ELPP_INTERNAL_DEBUGGING_OUT_INFO << ELPP_INTERNAL_DEBUGGING_MSG(internalInfoStre
 #  if (ELPP_COMPILER_GCC && !ELPP_MINGW && !ELPP_OS_OPENBSD)
 #    define ELPP_STACKTRACE 1
 #  else
-#      define ELPP_STACKTRACE 0
+#    define ELPP_STACKTRACE 0
+#    ifdef EASYLOGGING_CC
 #      if ELPP_COMPILER_MSVC
-#         pragma message("Stack trace not available for this compiler")
+#        pragma message("Stack trace not available for this compiler")
 #      else
-#         warning "Stack trace not available for this compiler";
+#        warning "Stack trace not available for this compiler";
 #      endif  // ELPP_COMPILER_MSVC
-#    define ELPP_STACKTRACE 0
+#    endif
```

Comment:
Why is the define moved around (as well as some random whitespace changes) ?

Reply:
I did the indentation of 2 white spaces.
The `define ELPP STACK TRACE 0` was defined twice inside the second else block, but one could not see it easily because of the original 4 white spaces.

Reply:
OK, looks right, thanks.

Reply:
I did the indentation of 2 white spaces.
The `define ELPP_STACKTRACE 0` was defined twice inside the second else block ([#L206](https://github.com/monero-project/monero/blob/782a84f7b490baa10afef94dd6954d6ab50d020b/external/easylogging%2B%2B/easylogging%2B%2B.h#L206) and [#L212](https://github.com/monero-project/monero/blob/782a84f7b490baa10afef94dd6954d6ab50d020b/external/easylogging%2B%2B/easylogging%2B%2B.h#L212)), but one could not see it easily because of the original 4 white spaces.